microsoft/onefuzz
# fix multi-tenant deployment

When deploying a multitenant instance there was a mismatch between the identifier uri used to configure the application registration and the value we use to authenticate the cli client. This caused #1269 

To test: 
1- deploy on a multi tenant domain 
2- send a onefuzz request

Expected result: The request should be successful

closes #1269
